src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebar.tsx

```diff
@@ -132,6 +132,13 @@ export const ArtworkSidebar: React.FC<ArtworkSidebarProps> = ({
         <ArtworkSidebarCommercialButtonsFragmentContainer artwork={artwork} />
       )}
 
+      {!isInAuction && !hasEnded && sale && !sale.isAuction && (
```

Comment:
This seems like it should be extracted into some named condition in Metaphysics?

Reply:
Oh yeah – this is messy. I'd like to confirm the logic first and then happy to extract it (or maybe it'll be simple enough to do here).